Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -657,6 +657,10 @@ public void Include()
         {
             var path = GetFullPath(GetCurrentPath());
 
+            // if path is root of drive in windows (C:) return C:\ instead.
+            //  Should no break anything unless file/directory names end with :
+            if (path[path.Length - 1] == ':') path += "\\";
```

Comment:
This is not a good solution for fixing this bug, you are basically fixing one bug and creating another. 

Tho from the sounds of the bug it should get fixed on the Eto.Forms side as well.

Reply:
Both Mono and .Net running on Windows throw a NotSupported Exception(https://msdn.microsoft.com/en-us/library/system.io.path.getfullpath(v=vs.100).aspx) if there is a ":" anywhere in the string that is not a drive specifier.

Please point me to the accepted Platform check for Windows.  I will add that to the code then the bug created will not exist.

The form is properly handling the exception.

Reply:
For a proper check you need to check if the Length of the path is 2 and first character is not `/`. Also the check should be moved inside `GetFullPath` method.